api/src/wfl/module/covid.clj

```diff
@@ -535,8 +535,7 @@
 (defn ^:private update-method-configuration!
   "Update METHODCONFIGURATION in WORKSPACE with snapshot reference NAME
   as :dataReferenceName via Firecloud.
-  Update executor table record ID with incremented METHODCONFIGURATIONVERSION.
-  Return EXECUTOR with incremented METHODCONFIGURATIONVERSION."
+  Update executor table record ID with incremented METHODCONFIGURATIONVERSION."
```

Comment:
optional nit: or just 
```suggestion
  Update executor table record `id` with incremented `methodConfiguration`."
```
? Sad we have a mixture of docstring styles now :(

Reply:
I did a sweep of this namespace and adjusted all docstrings to backtick parameter names.